src/main/java/uk/gov/digital/ho/hocs/info/domain/model/CaseDetailsField.java

```diff
@@ -34,5 +34,8 @@ public class CaseDetailsField implements Serializable {
     @Column(name = "props")
     private String props;
 
+    @Getter
```

Comment:
I'd like to understand the problem this solves, I have a ticket to remove this table - what I view to be a WCS specific code smell - and replace it with a readonly boolean on the field table, but this seems to suggest something more complex?

Reply:
That would work, this table is a smell, but this change allows this to be changed quicker than the planned column removal and repointing. I'd like to get this in, to then allow it to be changed again. 

There is more to these as this table is designed to specify what's in the unallocated view from what I can see but is not used by non-WCS and that has its solution. One to look into more holistic. 

While this ticket didn't require this, it's just Boy Scout to make it easier. Let's not block this ticket for that.

Reply:
not suggesting we don't block it, but it might not be in prod long :D 
there must be an actual order, the order they appear in the forms in their stages?

Reply:
It does something fancy like that in CS. This code it used to work on order by ID. Where people were dropping the table and reinserting everything.